driver.sh

```diff
@@ -75,12 +81,36 @@ setup_variables() {
       echo "Unknown ARCH specified!"
       exit 1 ;;
   esac
+  export ARCH=${ARCH}
 
   # torvalds/linux is the default repo if nothing is specified
   case ${REPO:=linux} in
-    "linux") owner=torvalds ;;
-    "linux-next") owner=next ;;
+    "linux")
+      owner=torvalds
+      tree=linux
+      branch=master ;;
+    "linux-next")
+      owner=next
+      tree=linux-next
+      branch=master ;;
+    "4-18")
+      owner=stable
+      tree=linux
+      branch=linux-4.18.y ;;
+    "4-14")
+      owner=stable
+      tree=linux
+      branch=linux-4.14.y ;;
+    "4-9")
+      owner=stable
+      tree=linux
+      branch=linux-4.9.y ;;
+    "4-4")
```

Comment:
Hopefully once #42 is merged, all of these could be combined into:

```
"4.4"|"4.9"|"4.14"|"4.18"|"4.19")
  owner=stable
  branch=linux-${REPO}.y ;;
```

Reply:
It's a good recommendation for minimizing duplication between existing targets.